Missing vendor directory causing docker build to fail

The command `docker build -t gonymizer .` fails with the message:
```
build github.com/aws/aws-sdk-go/aws: cannot load github.com/aws/aws-sdk-go/aws: open /tmp/gonymizer/vendor/github.com/aws/aws-sdk-go/aws: no such file or directory
The command '/bin/sh -c GOOS=linux GOARCH=amd64 CGO_ENABLED=0 GOFLAGS=-mod=vendor go build -v -ldflags '-w -extldflags "-static"' -o bin/gonymizer ./cmd/...' returned a non-zero code: 1
```

I had to run `go mod vendor` on my machine before running `docker build`. Is this expected?

If not, adding `RUN go mod vendor` to `Dockerfile` fixes the issue, but then I would have to `COPY` the source code together with `go.mod`.
Or I can remove the use of `go mod vendor` from the project.

What do you think?

Last but not least, thank you for writing this great package, it's really helpful.

I'm running into the same issue: 

```
docker build gonymizer --tag latest
Sending build context to Docker daemon  46.18MB
Step 1/14 : FROM golang:1.13-alpine as build
 ---> 69cf534c966a
Step 2/14 : ENV GO111MODULE=on
 ---> Using cache
 ---> 350477ed58df
Step 3/14 : RUN apk update && apk upgrade && apk add --no-cache bash git gcc go linux-headers musl-dev postgresql curl
 ---> Using cache
 ---> b9ba4ff780da
Step 4/14 : RUN mkdir -p /tmp/gonymizer/
 ---> Using cache
 ---> e5aaea3b4ae1
Step 5/14 : RUN mkdir -p /tmp/gonymizer/bin
 ---> Using cache
 ---> 99a0496261b8
Step 6/14 : WORKDIR /tmp/gonymizer/
 ---> Using cache
 ---> 0a7a554e4257
Step 7/14 : COPY go.mod .
 ---> Using cache
 ---> a98971661264
Step 8/14 : COPY go.sum .
 ---> Using cache
 ---> cd9878f330b1
Step 9/14 : RUN go mod download
 ---> Using cache
 ---> cecf931a348b
Step 10/14 : COPY . .
 ---> Using cache
 ---> eaad88cb9b0f
Step 11/14 : RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 GOFLAGS=-mod=vendor go build -v -ldflags '-w -extldflags "-static"' -o bin/gonymizer ./cmd/...
 ---> Running in ff7613dcfeaf
build github.com/aws/aws-sdk-go/aws/session: cannot load github.com/aws/aws-sdk-go/aws/session: open /tmp/gonymizer/vendor/github.com/aws/aws-sdk-go/aws/session: no such file or directory
build github.com/logrusorgru/aurora: cannot load github.com/logrusorgru/aurora: open /tmp/gonymizer/vendor/github.com/logrusorgru/aurora: no such file or directory
The command '/bin/sh -c GOOS=linux GOARCH=amd64 CGO_ENABLED=0 GOFLAGS=-mod=vendor go build -v -ldflags '-w -extldflags "-static"' -o bin/gonymizer ./cmd/...' returned a non-zero code: 1
make: *** [build-container] Error 1
```
How does this director get populated? I'd like to try a workaround and mount a volume for it from my local mac, if possible.

--- a/.dockerignore
+++ b/.dockerignore
@@ -1,0 +1,5 @@
+*
+!cmd
+!*.go
+!go.mod
+!go.sum

--- a/Dockerfile
+++ b/Dockerfile
@@ -2,21 +2,19 @@
 ### BUILD ###
 #############
 FROM golang:1.13-alpine as build
-ENV GO111MODULE=on
-RUN apk update && apk upgrade && apk add --no-cache bash git gcc go linux-headers musl-dev postgresql curl
-RUN mkdir -p /tmp/gonymizer/
+RUN apk update && apk upgrade && apk add --no-cache gcc musl-dev postgresql
 RUN mkdir -p /tmp/gonymizer/bin
 WORKDIR /tmp/gonymizer/
 COPY go.mod .
 COPY go.sum .
 RUN go mod download
 COPY . .
-RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 GOFLAGS=-mod=vendor go build -v -ldflags '-w -extldflags "-static"' -o bin/gonymizer ./cmd/...
+RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -v -ldflags '-w -extldflags "-static"' -o bin/gonymizer ./cmd/...
 
 ##########################
 ### Gonymizer Runtime  ###
 ##########################
 FROM golang:1.13-alpine as gonymizer
-RUN apk update && apk upgrade && apk add --no-cache postgresql curl
+RUN apk update && apk upgrade && apk add --no-cache postgresql
 
 COPY --from=build /tmp/gonymizer/bin/gonymizer /usr/bin/gonymizer

